opencog/atoms/core/NumberNode.h

```diff
@@ -54,7 +55,7 @@ class NumberNode : public Node
 	// the European comma as a decimal separator blows up the code.
 	static std::string double_to_string(double x)
 	{
-		std::string vs(std::to_string(x));
+		std::string vs = boost::lexical_cast<std::string>(x);
 		std::size_t found = vs.find(',');
```

Comment:
well, if you are going to do this, then you can remove lines 44 through 63, since lexical_cast ignores the locale, and won't have the problem described in the comments. i.e. just do this:
```
static std::string double_to_string(double x){
    return boost::lexical_cast<std::string>(x); }
```
```

Reply:
fixed